spec/lib/mhv/account_creation/service_spec.rb

```diff
@@ -16,11 +16,13 @@
     let(:account_creation_base_url) { 'https://apigw-intb.aws.myhealth.va.gov' }
     let(:account_creation_path) { 'v1/usermgmt/account-service/account' }
     let(:break_cache) { false }
+    let(:start_time) { Time.current }
 
     before do
       allow(Rails.logger).to receive(:info)
       allow(Rails.logger).to receive(:error)
       allow_any_instance_of(SignInService::Sts).to receive(:base_url).and_return('https://staging-api.va.gov')
+      Timecop.freeze(start_time)
```

Comment:
I was looking at this and trying to figure out how to get this to return a non `0.0` duration, but not being able to figure it out for `Timecop`. So I guess this does need to be mocked, and you can give it two times, which will mock it at the first value the first time, and the second value the second time:
`allow(Time).to receive(:current).and_return(start_time, end_time)`

This is what those `start_time` and `end_time` values look like:

`let(:start_time) { Time.zone.now }`
`let(:end_time) { start_time + 10.seconds }`

One caveat:
- Make sure you use `Time.zone.now` instead of `Time.current` elsewhere in the spec, otherwise it will count as one of these 'mocked' calls and won't work where you expect